plugins/spark/flytekitplugins/spark/task.py

```diff
@@ -76,12 +76,16 @@ def __init__(self, task_config: Spark, task_function: Callable, **kwargs):
         )
 
     def get_custom(self, settings: SerializationSettings) -> Dict[str, Any]:
+
         from flytekit.bin import entrypoint as _entrypoint
```

Comment:
can we delete this from here and move to the entrypoint path creation location. what i meant was that the defaulting should happen at the top level.

Reply:
done